views/template.njk

```diff
@@ -19,7 +19,7 @@
     <section class="section inner-content">
       <div class="container">
         <h1 class="title is-1">
-          Template {{template.getIdentifier()}}
+          {{template.getMetadata().getDisplayName()}} Template
```

Comment:
I think it would be useful to keep the id somewhere on the page (maybe under the page title?).

Reply:
This could happen as an additional PR.

Reply:
This is a good idea. I can put it here next to the hash.

<img width="962" alt="Screen Shot 2019-07-31 at 9 11 53 AM" src="https://user-images.githubusercontent.com/36460856/62214609-67776680-b373-11e9-8b2f-366bfdf70a3c.png">